MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -2536,16 +2270,15 @@ public int hashCode() {
     }
 
     public int getExperienceLevel(boolean secondary) {
```

Comment:
I'm not certain the logic for this method should stay here or be moved to the PersonnelRole Enum given how much it relies on the latter, but the heavy reliance on Personnel is why I've kept it here.